src/core/Akka.Cluster/ClusterHeartbeat.cs

```diff
@@ -26,16 +26,16 @@ internal sealed class ClusterHeartbeatReceiver : UntypedActor
     {
         // Important - don't use Cluster.Get(Context.System) in constructor because that would
         // cause deadlock. See startup sequence in ClusterDaemon.
-        private readonly Lazy<Cluster> _cluster;
```

Comment:
No real reason for a `Lazy<Cluster>` here since we're already receiving an instance in this actor's constructor at the time it's created.